modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -24,19 +25,25 @@ trap nagios_passive EXIT
 
 # Backup mongodb
 TIME="$(date +%s)"
-/usr/bin/mongodump -h $BACKUP_NODE -o $BACKUP_DIR --quiet
+/usr/bin/mongodump -h $BACKUP_NODE -o $BACKUP_DIR
 TIME="$(($(date +%s)-TIME))"
 printf "BACKUP COMPLETED IN: ${TIME}s\n"
 
 # Compress and encrypt backup
 TIME="$(date +%s)"
-(/bin/tar cvz $BACKUP_DIR | /usr/bin/gpg -e -o $BACKUP_DIR/$BACKUP_FILE -r $KEY_FINGERPRINT) &> /dev/null
+(/bin/tar cvz $BACKUP_DIR | /usr/bin/gpg -e -o $BACKUP_DIR/$BACKUP_FILE -r $KEY_FINGERPRINT)
 TIME="$(($(date +%s)-TIME))"
 printf "COMPRESSION AND ENCRYPTION FINISHED IN: ${TIME}s\n"
 
 # Upload backup to s3 bucket
 TIME="$(date +%s)"
-/usr/bin/envdir <%= @env_dir %>/env.d /usr/local/bin/s3cmd put $BACKUP_DIR/mongodump-* s3://$S3_BUCKET --quiet
+CURRTIME=$(date +%H%M)
+if [ $CURRTIME -ge 0000 -a $CURRTIME -le 0010 ]; then
```

Comment:
I like the intent but not a fan of hardcoding the time into the code. It might be better to pass in an argument in the cron job, for example `/usr/local/bin/mongodb-backup-s3 daily`, and then do a conditional around the argument: `if [ $1 == "daily" ]; then ...`